### PR TITLE
feat: Add link to attendee information in 2 UnTechCon posts

### PR DESCRIPTION
**Pull request by katjam**
I've been unable to verify locally (ruby broken) and unsure on wording and placement we wanted - but thought it was worth a PR anyway.

As a starting point.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.7.4
```
Comment (katjam) [inline .ruby-version:1]: Sorry - my editor did this on save, I can reset if you prefer as before.

**Comment by katjam** [comment]
image

image

**Review by aaaaargZombies — APPROVED** [review]
good improvement - CoC looks good too.

I know people were merging there own PRs but I'll do it as it's better for attendees to have a chance to see it sooner.

**Comment by katjam** [comment]
Thank you!

**Pull request merged**